internal/presentation/loop.go

```diff
@@ -68,9 +72,20 @@ func (g gameLoop) rotateRemainingTile(rotationType RotationType) GuiHandler {
 	return func(gui *gocui.Gui, view *gocui.View) error {
 		switch rotationType {
 		case RotateClockWise:
-			g.state.RotateRemainingTileClockwise()
+			g.board.RotateRemainingTileClockwise()
 		case RotateAntiClockWise:
-			g.state.RotateRemainingTileAntiClockwise()
+			g.board.RotateRemainingTileAntiClockwise()
+		}
+		return g.saveBoard()
+	}
+}
+
+func (g gameLoop) moveCurrentPlayerTo(line, row int) GuiHandler {
+	return func(gui *gocui.Gui, view *gocui.View) error {
+		if err := g.board.MoveCurrentPlayerTo(line, row); err != nil {
+			if err != model.ErrInvalidAction {
```

Comment:
I don't understand the rationale of this test. Why should you swallow invalid action errors?

Reply:
This is the same reason as previous comment, I will add a comment to provide the reason in the code.